package.json

```diff
@@ -29,7 +29,7 @@
     "lint": "vue-cli-service lint"
   },
   "dependencies": {
-    "barcode-detector": "^1.0.3",
+    "barcode-detector": "2.2.2",
```

Comment:
2.2.3 is released (with some fixes on the zing-cpp side), so let's use the latest version.
```json
"barcode-detector": "2.2.3",
```